README.md

```diff
@@ -2,4 +2,4 @@
 
 | Project                             | Link                                                                                                                                                                                 | Description                         |
 | ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------|
-| foo             | link                                                                                                                             | bar                            |
+| ODLM (operand deploymentlifecycle manager)            | [ODLM link](https://github.com/IBM/operand-deployment-lifecycle-manager)                                                                                                                             | ODLM is used to manage the lifecycle of for a group of operands.                          |
```

Comment:
How about delete `Link` column, and update `project` to have the hyper link?

Reply:
@gyliu513 I think so, I remove the link field.